dataset/dictionary/weapons.json5

```diff
@@ -709,46 +839,64 @@
   {
     en: "Fading Twilight",
     ja: "落霞",
+    zhCN: "落霞",
     pronunciationJa: "らっか",
     tags: [ "weapon", "bow" ],
   },
+  {
+    en: "Predator",
+    ja: "プレデター",
+    zhCN: "掠食者",
+    tags: [ "weapon", "bow" ],
+  },
```

Comment:
Sorry, can you remove this?
I don't want to add Aloy-related words because (at least Japanese) community is flamed when Aloy is announced and it reminds me negative emotion.

```suggestion
```

Reply:
Sorry to hear that. I will pay attention to it.

Reply:
Thanks for understanding 🙇